CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ ENHANCEMENTS:
 * resource/tfe_saml_settings: Add `private_key_wo` write-only attribute, by @uturunku1 ([#1660](https://github.com/hashicorp/terraform-provider-tfe/pull/1660))
 
 * resource/tfe_ssh_key: Add `key_wo` write-only attribute, by @ctrombley ([#1659](https://github.com/hashicorp/terraform-provider-tfe/pull/1659))
```

Comment:
Was the changelog item convention change intentional? At least I find the old one more readable.

Reply:
This was unintentional, good catch! Much appreciated ✨ 

We'll address this in a separate PR.